plans/example.fmf

```diff
@@ -0,0 +1,4 @@
+summary: Test the evaluation of llvm_sb macro
+discover:
+    how: shell
+    script: rpm --eval "%{llvm_sb}" | grep "llvm_snapshot_version:"
```

Comment:
```suggestion
    tests:
    - name: version check
      test: rpm --eval "%{llvm_sb}" | grep "llvm_snapshot_version:"
```

Reply:
@TomasTomecek thank you! I've incorporated your help.